docs/api-reference/contexts.md

```diff
@@ -155,52 +157,56 @@ The [Koa Context](https://github.com/koajs/koa/blob/master/docs/api/context.md)
 #### `handlerCtxt.invoke`
 
 ```typescript
-invoke<T>(targetClass: T, workflowUUID?: string): InvokeFuncs<T>
+invoke<T>(target: T, workflowUUID?: string): InvokeFuncs<T>
 ```
 
-Invoke a transaction or communicator.
+Invoke a transaction or communicator on the `target` class or configured instance.
 To invoke workflows, use [`invokeWorkflow`](#handlerctxtinvokeworkflow) or [`startWorkflow`](#handlerctxtstartworkflow) instead.
 The syntax for invoking function `fn` in class `Cls` with argument `arg` is:
 
 ```typescript
 const output = await handlerCtxt.invoke(Cls).fn(arg);
 ```
 
-You don't need to supply a context to the invoked function&#8212;the DBOS Transact runtime does this for you.
-You can optionally provide a UUID idempotency key to the invoked function.
+You aren't to supply a context to the invoked function&#8212;the DBOS Transact runtime does this for you.
```

Comment:
Minor style point, but I liked the old way better here

Reply:
The "need to" implied you could.  How about "You don't supply"?

Reply:
Yeah that's much better!